bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -103,6 +103,11 @@ public void textBoxKeyPress(final TextBoxWidget tbwe) {
         });
 
         _agsPub.subscribe((obs, oldOptions, newOptions) -> updateGuiding());
+
+        BagsManager.instance().addBagsStatusListener((obs, oldStatus, newStatus) -> {
+            if (obs == getContextObservation())
```

Comment:
Ugh... again, in this case, we want to compare references and not actual content.